syft/grid/utils/autoscale/gcloud.py

```diff
@@ -241,14 +239,15 @@ def create_cluster(
             network_interface={"network": "default", "access_config": {}},
             metadata_startup_script=f"""
                 #!/bin/bash
-                apt-get update
-                apt-get -y upgrade
-                sudo -i bash -c 'pip install notebook==5.7.8'
-                sudo -i bash -c 'pip install git+https://github.com/OpenMined/PyGridNode.git'
-                sudo -i bash -c 'echo Starting Node {name} \
-                joined with PyGridNetwork at {pygrid_network_address} & \
-                python -m gridnode --id={name} --port=80 \
-                --gateway_url={pygrid_network_address}'""",
+                sleep 30;
+                docker pull openmined/grid-node:production;
+                docker run \
+                -e NODE_ID={name} \
+                -e HOST="$(curl ifcongif.co)" \
```

Comment:
shouldn't be `ifconfig.co`?

Reply:
`curl ifconfig.co` gives the external IP, that's what we wanted right?

Reply:
What I'm saying is that it's written "ifcongif.co"

Reply:
got it 😬